apps/DisplayCluster/resources/MultiTouchGestureArea.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.0
+import DisplayCluster 1.0
+import "qrc:/qml/core/style.js" as Style
+
+MultiPointTouchArea {
```

Comment:
There must be a nice and easy way with Qt/QML/Test to write a unit test for this to verify that we receive gestures for different touch point inputs. Similar to the now removed DoubleTap test, but only based on QML.